src/SignalR/server/Core/src/HubConnectionContextOptions.cs

```diff
@@ -34,4 +34,9 @@ public class HubConnectionContextOptions
     /// Gets or sets the maximum parallel hub method invocations.
     /// </summary>
     public int MaximumParallelInvocations { get; set; } = 1;
+
+    /// <summary>
+    /// Gets or sets the maximum bytes to buffer per connection when using stateful reconnect.
+    /// </summary>
+    public long MessageBufferSize { get; set; } = 100_000;
```

Comment:
Personally I like having units in names, but if this is the name that was decided during API review, I won't push back.

Reply:
Oh wait, that's tomorrow 😆

Reply:
So does that mean you pushed back ;)